dm/common.go

```diff
@@ -27,6 +27,13 @@ type DataDetails struct {
 	Links   	Link      	`json:"links, omitempty"`
 }
 
+type FolderContents struct {
```

Comment:
Hub's DataDetails don't have an Included so I just made one specifically for FolderContents